testproj/manage.py

```diff
@@ -6,6 +6,10 @@
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "testproj.settings")
     try:
         from django.core.management import execute_from_command_line
+
+        sys.path.append(
+            os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'src'))
+        )
```

Comment:
This should not be required if you have drf_yasg installed (development mode or not).